library/globals.inc.php

```diff
@@ -1372,6 +1372,12 @@
       '',                               // default
       xl('Email address, if any, to receive emergency login user activation messages.')
     ),
+     'new_validate' => array(
```

Comment:
It's only a suggestion, you can ignore it.  I'm sure you understand the point.  Thanks.